apps/site/assets/css/_utilities.scss

```diff
@@ -79,6 +79,25 @@
   background-color: $brand-primary;
 }
 
+.u-focus--bus {
+  &:focus {
+    color: $black;
+  }
+}
+
+.u-focus--green-line,
+.u-focus--red-line,
+.u-focus--silver-line,
+.u-focus--blue-line,
+.u-focus--subway,
+.u-focus--mattapan,
+.u-focus--commuter-rail,
+.u-focus--unknown {
```

Comment:
This seems a little unnecessary. Would making _any_ text in here white by default (no need to call out each mode type, but fine to keep semantic class), and then just making the `--bus` the single override outlier, be more efficient?